cmd/podman/root.go

```diff
@@ -71,7 +71,7 @@ var (
 		DisableFlagsInUseLine: true,
 	}
 
-	logLevel       = "error"
+	logLevel       = "warn"
```

Comment:
Is this something we let the user toggle via containers.conf?

Reply:
Maybe, eventually although if we do this correct, I don't think they would want to, or ever need to.